src/main/java/com/gluonhq/substrate/util/ios/InfoPlist.java

```diff
@@ -100,6 +105,10 @@ public Path processInfoPlist() throws IOException {
         String appName = projectConfiguration.getAppName();
         String executableName = getExecutableName(appName, sourceOS);
         String bundleIdName = getBundleId(getPlistPath(paths, sourceOS), projectConfiguration.getMainClassName());
+        ReleaseConfiguration releaseConfiguration = projectConfiguration.getReleaseConfiguration();
+        String bundleName = Optional.ofNullable(releaseConfiguration.getBundleName()).orElse(appName);
```

Comment:
Instead of the Optional pattern, you might want to consider this:

    String bundleName = Objects.requireNonNullElse(releaseConfiguration.getBundleName(), appName);

Reply:
Hmm, ok. Done